### DISCUSSION TITLE
请问现在可以申请加入团队吗？

### DISCUSSION THREAD

**Discussion by W4J1e** [volantis-x/hexo-theme-volantis#565] [Q&A] [answered]
爱Hexo，爱volantis，爱不务正业的一名小菜。
才接触hexo一个多月，一直用的volantis主题，在群里帮助过其他朋友解答我遇到并解决了的问题，在社区分享过文章，在主题提交过pr，好像也没做别的事了。
blog：https://www.hin.cool ~

**Comment by xaoxuu** [comment, marked as answer]
已经邀请了，感谢支持，希望坚持下去哦🤝

**Reply by W4J1e** [reply]
我会继续学习的🤝
